cannot find package "kube-psp-advisor/advisor"

I tried to build kube-psp-advisor, but it failed.

```
$ pwd
/home/ubuntu/src/github.com/sysdiglabs/kube-psp-advisor
$ make build
+ build
./scripts/build
+ find . -type f -name *.go -not -path ./vendor/*
+ /home/ubuntu/bin/goimports -w ./kube-psp-advisor.go ./utils/utils.go ./advisor/report/report.go ./advisor/types/securityspec.go ./advisor/advisor.go ./advisor/processor/generate.go ./advisor/processor/get.go
+ uname+ tr [:upper:] [:lower:]
 -s
+ env GOOS=linux GOARCH=amd64 go build -a
kube-psp-advisor.go:5:2: cannot find package "kube-psp-advisor/advisor" in any of:
        /home/ubuntu/src/github.com/sysdiglabs/kube-psp-advisor/vendor/kube-psp-advisor/advisor (vendor tree)
        /home/linuxbrew/.linuxbrew/Cellar/go/1.11.5/libexec/src/kube-psp-advisor/advisor (from $GOROOT)
        /home/ubuntu/src/kube-psp-advisor/advisor (from $GOPATH)
Makefile:9: recipe for target 'build' failed
make: *** [build] Error 1
```

--- a/advisor/advisor.go
+++ b/advisor/advisor.go
@@ -3,9 +3,10 @@ package advisor
 import (
 	"encoding/json"
 	"fmt"
-	"kube-psp-advisor/advisor/processor"
-	"kube-psp-advisor/advisor/report"
 	"os"
+
+	"github.com/sysdiglabs/kube-psp-advisor/advisor/processor"
+	"github.com/sysdiglabs/kube-psp-advisor/advisor/report"
 
 	"k8s.io/api/policy/v1beta1"
 	k8sJSON "k8s.io/apimachinery/pkg/runtime/serializer/json"

--- a/advisor/processor/generate.go
+++ b/advisor/processor/generate.go
@@ -2,12 +2,13 @@ package processor
 
 import (
 	"fmt"
-	"kube-psp-advisor/advisor/report"
-	"kube-psp-advisor/advisor/types"
-	"kube-psp-advisor/utils"
 	"time"
 
-	"k8s.io/api/core/v1"
+	"github.com/sysdiglabs/kube-psp-advisor/advisor/report"
+	"github.com/sysdiglabs/kube-psp-advisor/advisor/types"
+	"github.com/sysdiglabs/kube-psp-advisor/utils"
+
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/api/policy/v1beta1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"

--- a/advisor/processor/get.go
+++ b/advisor/processor/get.go
@@ -1,12 +1,13 @@
 package processor
 
 import (
-	"kube-psp-advisor/advisor/types"
-	"kube-psp-advisor/utils"
 	"reflect"
 	"strings"
 
-	"k8s.io/api/core/v1"
+	"github.com/sysdiglabs/kube-psp-advisor/advisor/types"
+	"github.com/sysdiglabs/kube-psp-advisor/utils"
+
+	v1 "k8s.io/api/core/v1"
 	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 

--- a/advisor/report/report.go
+++ b/advisor/report/report.go
@@ -1,7 +1,7 @@
 package report
 
 import (
-	"kube-psp-advisor/advisor/types"
+	"github.com/sysdiglabs/kube-psp-advisor/advisor/types"
 )
 
 const (

--- a/kube-psp-advisor.go
+++ b/kube-psp-advisor.go
@@ -2,8 +2,9 @@ package main
 
 import (
 	"flag"
-	"kube-psp-advisor/advisor"
 	"path/filepath"
+
+	"github.com/sysdiglabs/kube-psp-advisor/advisor"
 
 	"k8s.io/client-go/util/homedir"
 )

